howto.md

```diff
@@ -0,0 +1,109 @@
+
+So, you started using the Rebble Web Services, but want to add the paid plan for weather and dictation. There are a few things to consider, before you order it:
+* On iOS, replying to text messages isn't possible. This is a limitation from iOS. So, dictation will NOT add spoken replies to text messages.
+* Weather services is solely for the native Pebble weather app and for weather info in timeline. Watchfaces and 3rd party apps that offer weather info, have their own means to fetch weather info. You don't need the Rebble paid plan for them.
+* Payment is handled by Stripe, which only offers payment by credit card.
+
+Still want it? Let's go!
+* Open the default browser on your phone and head to [Rebble Account](https://auth.rebble.io/account/).
+* **Very important: make sue you are signed in with the same account you used when you first made the switch to Rebble!**
+* Choose your plan ($3.00/month or $33/year).
+* You are forwarded to Stripe. Enter your card info and finish the payment.
+* You are redirected to the Rebble site. Voice / Weather subscription should now be **Active**.
+* In the last paragraph, tap the "repeat the initial setup" link. Then, follow the rest of the procedure for stap 2 above.
+
+In the Pebble app, under Apps, the app "The Weather Channel" should now be there. Tap the gear button to configure it to your needs.
+
+## Common issues
+
+### Cannot add watchface, store says its added, but it's not on the phone of watch
+
+Sometimes, the locker gets out of sync with reality. To get it back in sync, do this:
+* Start the default browser of your phone and head to [the Rebble app store on the web](https://apps.rebble.io/en_US/watchfaces?dev_settings=true).
+* Find the watchface or app and tap it.
+* On the bottom, tap "download pwb".
+* Open the file with the Pebble app.
+* The Pebble app opens, and asks to install the watchface or app. Tap "OK".
+
+Now, the watchface or app is installed, and the locker is no longer out of sync for this watchface. You might have to repeat this for every watchface or app that shows this behaviour.
+
+### On Android (9.0+), incoming calls are displayed on the watch as "Unknown caller"
+
+In order to fix "Unknown caller" or altogether missing notifications on incoming calls for Pebble paired with Android 9.0 devices, there is a workaround with Dialer for Pebble app:
+* Install [Dialer for Pebble](https://play.google.com/store/apps/details?id=com.matejdro.pebbledialer) companion app and a respective App for Pebble.
+* Go to your Pebble App > Notifications tab and Disable notifications for Phone Calls.
+* Go to Dialer for Pebble app on your phone and enable Call Popup for Incoming and Outgoing calls.
+* Make sure Dialer for Pebble app has system permissions to access Contacts, Cal Log, Dialer and SMS on your phone.
+
+This should fix the issue. In case if Dialer for Pebble works inconsistently, make sure it runs in background both on your Pebble and on you Phone.
```

Comment:
```suggestion
This should fix the issue. In case if Dialer for Pebble works inconsistently, make sure it runs in background both on your Pebble and on your Phone.
```

might also be worth explaining how to inspect *Background Apps* on the watch?

Reply:
Done.